tasks/adduser.yml

```diff
@@ -1,5 +1,5 @@
 ---
-- name: adduser.conf
+- name: "overwrite adduser.conf"
```

Comment:
why the quotes?

Reply:
Two reasons: In this context quotes lets us know it's been edited, and also if some special character is used in the name the yaml parser won't attempt to parse it.